src/rsp_restspawner/spawner.py

```diff
@@ -181,6 +181,10 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         # Holds the events from a spawn in progress.
         self._events: list[SpawnEvent] = []
 
+        # Triggers used to notify listeners of new events. Each listener gets
+        # their own trigger.
```

Comment:
"its" -- "each" is singular.  "Their" would be correct if the listener's pronouns are "they/them" but as a non-sentient bit of code, I am comfortable with "it".